Source/Charts/Data/Implementations/Standard/ChartDataSet.swift

```diff
@@ -291,15 +272,15 @@ open class ChartDataSet: ChartBaseDataSet
         rounding: ChartDataSetRounding) -> Int
     {
         var low = 0
-        var high = _values.count - 1
+        var high = values.count - 1
```

Comment:
do you mean we shall change all `.count` into `.endIndex` in the future after collection adoption? Or just some.

Reply:
We should use `count` when we actually want to know how many of something there is, and `endIndex` when we mean the end of the collection. I'll double check, but assuming this is like the other method implementation, this should be `endIndex`.

I'll review the PR again to make sure I'm consistent.

Reply:
Sure, I don't mean you need to change others. Just to clarify what we should use. I checked the method names and they don't reflect clearly count or boundary. Sometimes we don't clearly which to use. I will wait.

Reply:
Are we good to go on this one then?

Reply:
just one about the observer. I can't see it?